src/api/goods.ts

```diff
@@ -10,10 +9,13 @@ export function getAll(): Promise<Good[]> {
 
```

Comment:
When you make a request to the server, don't forget to handle possible errors
try to add `catch` to `getAll`